ansible.cfg

```diff
@@ -0,0 +1,23 @@
+[defaults]
+forks = 50
+
+# SSH timeout
+timeout = 30
+host_key_checking = False
+
+roles_path = ./roles
+library = ./library
+module_utils = ./module_utils
+
+scp_if_ssh = True
+force_color = 1
+
+retry_files_enabled = False
+
+[paramiko_connection]
+record_host_keys = False
+
+[ssh_connection]
+pipelining = True
+ssh_args = -o ControlMaster=auto -o ControlPersist=60s
+control_path = %(directory)s/%%h-%%r
```

Comment:
These should be set in your local environment, as opposed to here. 
IMHO, this file should be removed